jest.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  roots: ["<rootDir>/ui/src"],
+  transform: {
+    "^.+\\.ts$": "ts-jest",
+  },
+  testRegex: "(/__tests__/.*|(\\.|/)(test))\\.ts$",
```

Comment:
If we control the test regex, can we avoid the `__` convention?

Reply:
Genau. It's not the prettiest convention but it is immediately recognizable, so maybe we leave it? Open to thinking about a better directory name